java/src/main/java/com/microsoft/graph/bulk/impl/BulkGremlinObjectMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.graph.bulk.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.microsoft.graph.bulk.impl.model.GremlinEdge;
+import com.microsoft.graph.bulk.impl.model.GremlinVertex;
+
+public final class BulkGremlinObjectMapper {
+    private BulkGremlinObjectMapper() {
+        throw new IllegalStateException("Utility class, should not be constructed");
+    }
+
+    /**
+     * Defines an ObjectMapper that has the Serialization Modules for both GremlinVertex and GremlinEdge classes
+     */
+    public static ObjectMapper getBulkGremlinObjectMapper() {
+        var mapper = new ObjectMapper();
```

Comment:
for the object mapper, maybe change to initialize only once?

Reply:
It is only called once though, see the UploadWithBulkLoader method on the com.microsoft.graph.bulk.sample.UploadWithBulkLoader<V, E> class. 

Intent is to allow the user to opt into a different object mapper, but provide a sample if they choice to. 

Not entirly sure what an alternative approach would be, that doesn't allow the user to create more than one of these, and still allow them to provide an alternative object mapper for different serialization approaches.